airflow/config_templates/default_airflow.cfg

```diff
@@ -208,6 +208,10 @@ max_num_rendered_ti_fields_per_task = 30
 # On each dagrun check against defined SLAs
 check_slas = True
 
+# Path to custom XCom class that will be used to store and resolve operators results
+# Example: xcom_class = path.to.CustomXCom
+xcom_class = airflow.models.xcom.BaseXCom
```

Comment:
How about `xcom_backend` similar to our `secrets_backend` or `lineage_backend` just for consistency?